modules/core/src/main/java/org/apache/ignite/configuration/ClientConnectorConfiguration.java

```diff
@@ -592,6 +604,27 @@ public ClientConnectorConfiguration setThinClientConfiguration(ThinClientConfigu
         return this;
     }
 
+    /** @return Session outbound message queue limit. */
+    public int getSessionOutboundMessageQueueLimit() {
+        return sesOutboundMsgQueueLimit;
+    }
+
+    /**
+     * Sets Client session outbound message queue limit. Limits the number of messages waiting to be sent from the
+     * server side to particular thin client. If the specified limit is exceeded, corresponding thin client connection
```

Comment:
In general, `ClientConnectorConfiguration` related not only to thin client, but also for jdbc and odbc, so perhaps it's better to use just 'client' in javadoc instead of 'thin client'

Reply:
Fixed.